layouts/almalinux-day-tokyo-2026/single.html

```diff
@@ -78,39 +78,60 @@ <h2 style="font-size: 50px; letter-spacing: 0.80px; font-weight: 600; margin-bot
+          style="background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.7)), url('/landingpages/aldt-2026/aldt-2026-header.jpg');"
+        >
           <div class="container al-py-lg">
             <div class="topic_item_1" style="padding-top: 15%;">
               <h1>{{ i18n "AlmaLinux Day: Tokyo" }}</h1>
               <h3>
                 {{ i18n "January 23rd, 2026" }}
               </h3>
               <div>
-				  {{ i18n "Fujitsu Uvance Kawasaki Tower" }}<br />
-				  {{ i18n "Uvance Innovation Studio, 26th floor" }}<br />
-				  {{ i18n "JR Kawasaki Tower" }}<br />
-				  {{ i18n "1-5 Omiyacho" }}<br />
-				  {{ i18n "Saiwai-ku" }}<br />
-				  {{ i18n "Kawasaki City" }}<br />
-				  {{ i18n "Kanagawa Prefecture" }}<br />
+                {{ i18n "Fujitsu Uvance Kawasaki Tower" }}<br />
+                {{ i18n "Uvance Innovation Studio, 26th floor" }}<br />
+                {{ i18n "JR Kawasaki Tower" }}<br />
+                {{ i18n "1-5 Omiyacho" }}<br />
+                {{ i18n "Saiwai-ku" }}<br />
+                {{ i18n "Kawasaki City" }}<br />
+                {{ i18n "Kanagawa Prefecture" }}<br />
               </div>
             </div>
             <div class="topic_item_2" style="padding-top: 30px;">
-              <a href="https://almalinux.connpass.com/event/377157/" class="btn btn-lg px-4 me-md-2 al-cta-green"><i class="bi bi-box-arrow-in-right al-index-community-icon"></i>  {{ i18n "Reserve your spot!" }}</a>
+              <a
+                href="https://almalinux.connpass.com/event/377157/"
+                class="btn btn-lg px-4 me-md-2 al-cta-green"
+                ><i
+                  class="bi bi-box-arrow-in-right al-index-community-icon"
+                ></i>
+                {{ i18n "Reserve your spot!" }}</a
+              >
             </div>
           </div>
         </div>
         <div class="al-index-register">
           <div class="container pt-4 px-3">
-            <h1>{{ i18n "Register early to find all of the day's joys together!" }}</h1>
+            <h1>
+              {{ i18n "Register early to find all of the day's joys together!" }}
+            </h1>
             <div>
-              {{ i18n "The details of this event are still being finalized, but you can find a preview of the schedule on the Japan AlmaLinux User's group page." }} <a href="https://almalinux.connpass.com/event/377157/">AlmaLinux Day Tokyo 2026</a>
+              {{ i18n "The details of this event are still being finalized, but you can find a preview of the schedule on the Japan AlmaLinux User's group page." }}
 
-                <a href="https://almalinux.connpass.com/event/377157">
-                  <div class="d-flex justify-content-center pt-5 px-3" >
-                    <img loading="lazy" src="/blog-images/2025/tokyo-2026.png" alt="CloudLinux" class="al-backer-logo" />
-                  </div>
-                </a>
+              <a href="https://almalinux.connpass.com/event/377157/"
```

Comment:
@alexiri why does prettier do this, and more importantly can we make it stop? This breaking up makes the code very difficult to parse.